doc/funkspec.md

```diff
@@ -34,21 +34,50 @@
+* Könyvből való szó tanulás:
+    * Mai diákok nem preferálják a könyvből való tanulást
+    * Könyvet magukkal kell hordaniuk a tanulóknak sőt ha többet akarnak tanulni akkor szótárt kell venniük
+    * Könyvet eldobják ha már nem kell így hulladék keletkezik
+    * Nem elég reszponzív és interaktív
+* Tanuló önellenőrzése külön papíron:
+    * Ez a módszer papír és toll igényes
+    * Önellenőrzés után hulladék keletkezik
+    * Sok időt elvesz az, hogy a tanuló felírj a szavakat
 ---
 ## 6. Igényelt üzleti folyamatok modellje
-* Azért hogy egyszerűbbé tegyük a diákok angol tanulását, létrehozunk egy programot ami a mai kornak megfelelő és a legtöbb böngészőn megfelelően tud megjelenni illetve futni. A diákoknak sokkal egyszerűbb lesz a tanulás hiszen a program segítségével egyszerre tanulhatnak illetve ellenőrizhetik a tudásukat addig amíg nem érzik úgy, hogy a megtanult szavak jól mennek és a tudásuk biztos. Illetve a diákoknak nem kell a könyvet elővenniük és azt átlapozva előkeresniük az adott szavakat és így ha esetleg iskolában hagyják a könyveiket akkor is tudnak tanulni. Illetve ha esetleg új szavakat szeretne megtanulni azt fel tudja vinni a rendszerbe, és ez által a diákok új szavakat is fognak tudni majd tanulni.
+* Online elérhető, ingyenes rendszer létrehozása
+    * Webes megjelenés mind mobilról (hordozhatóság), mind pedig számítógépről
+* Átlátható design kialakítása a weboldalon
+    * Dobozok alkalmazása
+    * Elegáns design kialakítása
+    * Letisztult , sima, de elegáns színű háttér
+* Egyszerű betűtípus és betűszín alkalmazása
+    * A weboldalon lévő adatok szerkeszthetővé tétele/Könnyű szerkeszthetőség
+    * A már felvett feladatok nevének szerkeszthetősége
+    * A már nem kívánatos feladatok törölhetősége
 ---
 ## 7. Követelménylista
-|    Modul    |      ID     |           Név          |   V.   |                                                 Kifejtés                                                      |
-| :---        |    :----:   |          :---:         | :---:  |                                                  :---:                                                        |
-| Jogosultság |      K1     | Jogosultsági szintek   |  1.0   |                     Felhasználó/vendég: feladat teljesítése, új szavak bevitele ; Admin: Új szavak bevitele                       |
-| Feladattípus|      K2     |          Teszt         |  1.0   |                         Teszthez hasonló feladat. Hibás, hibátlan a teszt kitöltés                            |
+|Követelmény azonosító|      		      Leírás                      |
+|        :----:       |			      :----:		          |
+|         K01 	      |  Felhasználó/vendég tudja teljesíteni a feladatot |
+|	  K02 	      |  Admin tudjon új szavakat bevinni		  |
```

Comment:
A felhasználó is tudjon bevinni, külön admin fiókot szerintem a kis projekthez túl komplex lenne